docs/source/array-overview.rst

```diff
@@ -6,10 +6,25 @@ algorithms, cutting up the large array into many small arrays. This lets us
 compute on arrays larger than memory using all of our cores. We coordinate these
 blocked algorithms using dask graphs.
 
-For a live tutorial in your browser,
-visit our Binder_.
+Design
+------
 
-.. _Binder: http://mybinder.org/repo/dask/dask-examples/dask-array-basics.ipynb
+.. image:: images/dask-array-black-text.svg
+   :alt: Dask arrays coordinate many numpy arrays
+   :align: right
+
+Dask arrays coordinate many NumPy arrays arranged into a grid.  These
+arrays may live on disk or on other machines.
```

Comment:
Same comment as the dataframe docs:

> These arrays may live on disk or on other machines.

Is this referring to the serialized arrays on disk, or after being loaded as Dask arrays, or...?